src/SFML/Graphics/View.cpp

```diff
@@ -87,6 +87,13 @@ void View::setViewport(const FloatRect& viewport)
 }
 
 
+////////////////////////////////////////////////////////////
+void View::setScissor(const FloatRect& scissor)
+{
+    m_scissor = scissor;
```

Comment:
Should we assert that this float rect position and size are within [0, 1]? It would be friendly to users to check this for them.